CHANGELOG.md

```diff
@@ -17,7 +17,9 @@ All notable changes to the `python-domino` library will be documented in this fi
 * Fixed issues with import from domino_data
 * Updated nbconvert dependency version to 6.3.0
 * updated request user-agent to python-domino/{version}
+* Updated dependency management
 * Changed file handling for `files_upload`
+* removed requirements.txt in favor setup.py
```

Comment:
I think it's nice to have a requirements.txt in addition to setup.py to list all dependencies. Why do you want to remove it?

Reply:
In order to have a singular source of dependencies, otherwise, we risk having one being changed, leaving the other unchanged, which can then lead to dependency issues.